perf/Manifest.toml

```diff
@@ -1,7 +1,8 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.7.3"
+julia_version = "1.8.0"
```

Comment:
Why are we partially updating here?

Reply:
I needed to add `DataStructures.jl`, and that updated the manifest. Should I install Julia 1.7.3 and redo this?

Reply:
Yeah, maybe do this with julia 1.7.3, we still can't upgrade yet (see https://github.com/CliMA/ClimaTimeSteppers.jl/pull/55)